strawberry/subscriptions/protocols/graphql_ws/handlers.py

```diff
@@ -82,6 +84,9 @@ async def handle_message(
             await self.handle_stop(message)
 
     async def handle_connection_init(self, message: OperationMessage) -> None:
+        payload = cast(ConnectionInitPayload, message.get("payload"))
+        self.connection_params = payload or dict()
```

Comment:
Same as with the other protocol. The payload/connectionParams are allowed to be `null`/`None`. I think it would be best to preserve that and don't make it an empty dict instead.